tests/CMakeLists.txt

```diff
@@ -98,25 +98,30 @@ foreach(file ${ALL_TESTS})
     set(TEST_CONFIGURATIONS "slow")
   endif()
 
+  set(TEST_NAME ${file})
```

Comment:
Should this be TEST_EXPECTATION_NAME for consistency?

Reply:
done.